src/css/elements/button.css

```diff
@@ -0,0 +1,18 @@
+.button {
+  font-weight: var(--font-medium);
+  font-size: var(--text-md);
+  color: var(--color-primary-B500);
+  line-height: 24px;
+  padding: 12px;
+  border-radius: 8px;
+  border: 1px solid var(--color-primary-B400);
+}
```

Comment:
- [ ] You seemed to have missed implementing button interactions in your Portfolio as was required. You can refer to the image below to see what interactions you have to implement from the Figma template

![image](https://user-images.githubusercontent.com/90556221/164725955-f22ea541-6781-46e5-bcf7-9e997e69a91a.png)

To do so, you can use different CSS selectors on your buttons. For example, you can use the [`:hover`](https://www.w3schools.com/cssref/sel_hover.asp) selector to add hove effects to your elements

Reply:
### Thank you for your comment.
issue fixed here [Add button variants](https://github.com/teckim/portfolio/pull/2/commits/50e24c255599cebcf41adfea99399f68e449a95e)